mugoh/qstioner
# #163697894 Display meetups on the UI pages with GET requests

#### What does this PR do?
Render meetup data to pages with meetup elements
#### Description of Task to be completed?
- A newly created meetup is visible on pages that show present meetups.
- Remove dummy meetup displays on the admin dashboard.

#### How should this be manually tested?
Ensure the server is running. Follow the steps given [here](https://github.com/hogum/qstioner-v2) to do so.
1. Clone the repo in your machine
``` shell
$ git clone https://github.com/hogum/qstioner
```
2. Navigate to the directory `qstioner` where you have the repo cloned
``` shell
$ cd qstioner
```
3. Checkout this Feature branch
``` shell
$ git checkoutft-display-meetups-163697894
```
4. Open the `index.html` file
``` shell
$ browser-name index.html
```
or
Simply  load up on your preferred browser

 Created meetups should be visible on the meetups display area.
A new meetup can be created on the `create_meetup.html` page, to ascertain it shows up in the index page and the admin dashboard.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#163697894](https://www.pivotaltracker.com/story/show/163697894)
